act-rules-format.bs

```diff
@@ -21,6 +21,8 @@ This format is intended to enable a consistent interpretation of how to test for
 
 Describing how to test certain accessibility requirements will result in accessibility tests that are transparent, with test results that are reproducible. The Accessibility Conformance Testing (ACT) Rules Format defines the requirements for these test descriptions, known as Accessibility Conformance Testing Rules (ACT Rules).
 
+The ACT Rules Format defines the requirements and rule structure for the types of information each rule needs to include to be called an ACT Rule. The structure of the ACT rule is defined in the [=act-rule-structure=] section. Each ACT Rule also contains a plain language description of the type of content under test, the test to perform, and the expected result.  Where the test result affects conformance, the rule documents which particular requirement the test is for. The resulting outcomes from the test can be used to help determine conformance or non-conformance to the requirement. Test cases are also written as part of the ACT rule to provide a way to verify that implementations of the rule can successfully determine the expected outcomes.
```

Comment:
I was looking for a bit more of a conceptual to start out on, I think. 

E.g. "An ACT Rule is a plain language description of how to test for an accessibility requirement for a specific type of content. An ACT Rule describes what kind of content it applies to and which condtions must be true about the applicable elements for them to pass the rule..." (and then some more detailed stuff)

Reply:
@annethyme How's the short paragraph I added?  Need more?